src/pages/test-report/components/competency/competency.ts

```diff
@@ -46,15 +41,19 @@ interface CompetencyState {
 @Component({
   selector: 'competency',
   templateUrl: 'competency.html',
-  providers: [HammerProvider],
 })
 export class CompetencyComponent {
 
   @Input()
   competency: Competencies;
 
-  @ViewChild('competencyButton')
-  button: ElementRef;
+  touchStateDelay: number = 100;
+
+  touchState: boolean = false;
+  rippleState: boolean = false;
+
+  rippleTimeout: any;
```

Comment:
Apparently `setTimeout` returns a positive integer value as the timeoutId - https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/setTimeout#Return_value - So wonder if setting the type as `number` would work

Reply:
Setting it as number gives a TypeScript error :(